ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -257,6 +257,7 @@ public PendingTransactions getPendingTransactions() {
               transaction.getGasLimit(), chainHeadBlockHeader.getGasLimit()));
     }
 
+    // TODO: this is where we would use the private state to do the validation against
```

Comment:
This TODO isn't related to this change. We should remove it.

Reply:
Done!